server/athenian/api/models/state/models.py

```diff
@@ -284,3 +283,39 @@ class LogicalRepository(create_time_mixin(created_at=True, updated_at=True), Bas
     repository_id = Column(BigInteger(), nullable=False)
     prs = Column(JSON, nullable=False, default={}, server_default="{}")
     deployments = Column(JSON, nullable=False, default={}, server_default="{}")
+
+
+class Goal(create_time_mixin(created_at=True, updated_at=True), Base):
+    """Goal - A metric target for a team."""
+
+    __tablename__ = "goals"
+
+    id = Column(Integer(), primary_key=True)
+    account_id = Column(
+        Integer(),
+        ForeignKey("accounts.id", name="fk_goal_account"),
+        nullable=False,
+    )
+    template_id = Column(Integer(), nullable=False)
+    valid_from = Column(Date, nullable=False)
+    expires_at = Column(Date, nullable=False)
```

Comment:
```suggestion
    valid_from = Column(TIMESTAMP(timezone=True), nullable=False)
    expires_at = Column(TIMESTAMP(timezone=True), nullable=False)
```
(we always require the explicit UTC timezone everywhere)

Reply:
I chose the "date" type since that is what the UI handles, and that has no timezone.
Should we have a timestamp instead ? Some related decisions:
- in API we truncate to day anyway ?
- when we receive `{validFrom: 2000-01-01, expiresAt: 2001-04-01}`, should we save `expires_at: 2001-04-01T23:59:59.99999` ? that would be conceptually right

Reply:
We kinda implicitly expect UTC in the UI. @akbarik @alex-athenian note: all the times that the webapp sends us must be in UTC.
Regarding `expires_at`, we can save `2001-04-02 00:00:00` (+1 day). That's how our filters work, I think.

Reply:
So `2001-04-02 00:00:00` in DB is "2001-04-01" in API (for `expiresAt`) or API will express the full timestamp ?

Reply:
The first one is correct.

Reply:
done, changed fields as timestamp with time zone